pkg/network/tracer/tracer_test.go

```diff
@@ -25,6 +25,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/network/tracer/testutil/testdns"
```

Comment:
Belongs in the last group of imports below

Reply:
Is there a goimports task we have to sort these? I ran all my changes through goimports and this is what came out.

Reply:
https://github.com/DataDog/datadog-agent/blob/main/docs/dev/imports.md